test/integration/asset-integration.ts

```diff
@@ -146,6 +156,7 @@ describe('Asset api', function () {
               file: '<svg xmlns="http://www.w3.org/2000/svg"><path fill="red" d="M50 50h150v50H50z"/></svg>',
             },
           },
+          description: {},
```

Comment:
`description` is mandatory on the type (where the definition was missing previously).